print-ntp.c

```diff
@@ -516,3 +530,63 @@ p_poll(netdissect_options *ndo,
 		ND_PRINT(" (1/%us)", 1U << -poll_interval);
 }
 
+/* Prints an NTPv4 extension field */
+static void
+p_ntp_ef(netdissect_options *ndo, u_int type, u_int length, const u_char *ef_body)
+{
+	switch (type) {
+	case 0x0104:
+		ND_PRINT("\n\t  Unique Identifier");
+		break;
+	case 0x0204:
+		ND_PRINT("\n\t  NTS Cookie");
+		break;
+	case 0x0304:
+		ND_PRINT("\n\t  NTS Cookie Placeholder");
+		break;
+	case 0x0404:
+		ND_PRINT("\n\t  NTS Authenticator and Encrypted Extension Fields");
+		break;
+	case 0x2005:
+		ND_PRINT("\n\t  Checksum Complement");
+		break;
+	default:
+		ND_PRINT("\n\t  ?");
+	}
+
+	ND_PRINT(" (0x%04x), length %u", type, length);
+
+	if (ndo->ndo_vflag > 2)
+		hex_print(ndo, "\n\t    ", ef_body, length - 4);
+}
+
+/* Prints list of extension fields per RFC 7822 */
+static u_int
+p_ext_fields(netdissect_options *ndo, const u_char *cp, u_int length)
+{
+	const struct ntp_extension_field *ef;
+	u_int ef_type, ef_len, efs_len;
+	int first_ef;
+
+	first_ef = 1;
+	efs_len = 0;
+
+	while (length - efs_len > 24) {
```

Comment:
Why `24`? A comment would be useful to explain.

Reply:
I added a comment explaining the constant and changed the condition to `>= 28` to make it more obvious.